pkg/ddevapp/snapshot_test.go

```diff
@@ -44,6 +44,10 @@ func TestDdevSnapshotCleanup(t *testing.T) {
 	snapshotName, err := app.Snapshot(t.Name() + "_1")
 	assert.NoError(err)
 
+	// Provide a second snapshot that should not be deleted and whose filename will be lexicographically earlier (Issue #6694).
```

Comment:
Let's use the full URL here because IDE won't understand what `#6694` is.

```suggestion
	// Provide a second snapshot that should not be deleted and whose filename will be lexicographically earlier (See https://github.com/ddev/ddev/issues/6694).
```

Reply:
should this be an amend or a new commit?

Reply:
Doesn't matter, we squash everything in one commit when it lands in master.